pandamium_datapack/data/pandamium/functions/on_join.mcfunction

```diff
@@ -15,13 +18,13 @@ scoreboard players enable @s home
 scoreboard players enable @s sethome
 scoreboard players enable @s show_cooldown
 scoreboard players enable @s show_playtime
-scoreboard players enable @s particles
 scoreboard players enable @s tpa
 scoreboard players enable @s tpa_accept
 
-
-scoreboard players enable @s top_playtime
-scoreboard players enable @s top_votes
+execute if score @s gameplay_perms matches 3.. run scoreboard players enable @s pose
```

Comment:
People without the donator rank should be able to see that those triggers exist but get an error message when trying to use it, we can't rely on everyone being on Discord and knowing that this stuff even exists

Reply:
This also applies to `pose`, `top_playtime` and `top_votes` (I'm pretty sure top playtime/votes worked like this before already?)